AutoDefineAddon/autodefine.py

```diff
@@ -114,20 +82,20 @@ def get_definition_force_phonetic_transcription(editor):
 def validate_settings():
     # ideally, we wouldn't have to force people to individually register, but the API limit is just 1000 calls/day.
 
-    if PREFERRED_DICTIONARY != "COLLEGIATE" and PREFERRED_DICTIONARY != "MEDICAL":
-        message = "Setting PREFERRED_DICTIONARY must be set to either COLLEGIATE or MEDICAL. Current setting: '%s'" \
-                  % PREFERRED_DICTIONARY
+    if not settings.PREFERRED_DICTIONARY in ["COLLEGIATE", "MEDICAL", "SPANISH"]:
+        message = "Setting PREFERRED_DICTIONARY must be set to COLLEGIATE, MEDICAL or SPANISH. Current setting: '%s'" \
+                  % settings.PREFERRED_DICTIONARY
```

Comment:
can we pull out `["COLLEGIATE", "MEDICAL", "SPANISH"]` into a set as a global variable somewhere? Then we can re-use it in the message and potentially refer to it elsewhere in code as well